### ISSUE TITLE
Pipeline controller execute_remotely inconsistent

### ISSUE THREAD

**Issue by valeriano-manassero** [allegroai/clearml#273]
Hi,
if a pipeline controller is created with
```
task.execute_remotely()
```
The task is correctly set in draft mode for later usage on UI.
Unfortunately this task will not work since Pipeline Configuration Object is not created and this is leading to an error when enqueueing generating an empty object:
```
WARNING - Could not retrieve remote configuration named 'Pipeline'
Using default configuration: {}
```
AS a side note, also `pipeline` tag is not created until the task is enqueued .

**Comment by bmartinn** [comment]
Thanks @valeriano-manassero 

The bug is verified. A bit of background, the Pipeline Controller stores the entire pipeline n the Task configuration.

This way you can clone a pipeline from the UI, edit it and launch it again, without needing to change the original code!

That said, in your example the pipeline task is stopped (`execute_remotely` called) before the pipeline controller stored the pipeline configuration to the Task.

I'll update here once a fix is out (should not take long ;)

**Comment by bmartinn** [comment]
@valeriano-manassero updating here that the bug has been fixed in 0.17.4
``` bash
pip install clearml==0.17.4
```

**Comment by valeriano-manassero** [comment]
I can confirm this is now fixed.
